src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -3912,6 +3915,11 @@ int RGWRados::fetch_remote_obj(RGWObjectCtx& obj_ctx,
   set_mtime_weight.high_precision = high_precision_time;
   int ret;
 
+  auto trace = tracing::rgw::tracer.add_span("fetch_remote_obj", dest_obj.get_trace());
+  trace->SetAttribute(tracing::rgw::OBJECT_NAME, dest_obj.key.name);
+  trace->SetAttribute(tracing::rgw::BUCKET_NAME, dest_bucket_info.bucket.name);
```

Comment:
the trace should probably include the object's instance id and the bucket's tenant